### DISCUSSION TITLE
Why is the voxel size is changed when using basic_unet

### DISCUSSION THREAD

**Discussion by resinalec** [Project-MONAI/MONAI#8200] [Q&A]
Hello everyone,

I have been experimenting with basic_unet and I have found that when I check the voxel size of the output image it seems to have been changed.

For instance my input images have the voxel size1.6x.1.6x1.6 and my output images have the voxel size 1x1x1.

I would be really grateful if someone could explain why is this the case

**Comment by KumoLiu** [comment]
Hi @resinalec, could you share a code snippet to help me understand what’s happening? If you are using `MetaTensor`, the affine won't be changed after net.

**Reply by resinalec** [reply]
Yes thank you I wasn't correctly using the affine matrix, I just assigned the output image the affine matrix of my input and it solves the confusion that I had. Also I am normalising my input images between 0 and 1 for this. Is this the correct approach.

`def normalize_image(image):
    return (image - image.min()) / (image.max() - image.min())`
